pages/notifications.js

```diff
@@ -16,41 +16,83 @@ import Me from '../components/Auth/Me'
 import TokenAuthorization from '../components/Auth/TokenAuthorization'
 import MacNewsletterSubscription from '../components/Auth/MacNewsletterSubscription'
 
-import { DEFAULT_TOKEN_TYPE } from '../components/constants'
+import MdClose from 'react-icons/lib/md/close'
+
+import {
+  DEFAULT_TOKEN_TYPE,
+  HEADER_HEIGHT,
+  HEADER_HEIGHT_MOBILE
+} from '../components/constants'
+
 import {
   CURTAIN_MESSAGE, CDN_FRONTEND_BASE_URL
 } from '../lib/constants'
 
 import {
-  Interaction, NarrowContainer, Logo, linkRule, RawHtml, mediaQueries, Button
+  Interaction, NarrowContainer, Logo, linkRule, RawHtml, mediaQueries, colors, Button
 } from '@project-r/styleguide'
 
+const LOGO_HEIGHT = 28.02
```

Comment:
Structure: Might this and `LOGO_HEIGHT_MOBILE` best be placed into a "constants" file?